manifests/globals.pp

```diff
@@ -16,9 +16,14 @@
   $ipv6                  = undef,
   $bind_ip               = undef,
 
-  $version               = undef,
-
-  $manage_package_repo   = undef,
+  $version               = fact('os.distro.codename') ? { # Debian 10 doesn't provide mongodb 3.6.
+    'buster' => '4.4.8',
+    default  => undef
+  },
+  $manage_package_repo   = fact('os.distro.codename') ? { # Debian 10 doesn't provide mongodb packages. So manage it!
+    'buster' => true,
+    default  => undef
+  },
```

Comment:
That mean that the same configuration on some nodes install the version packaged for the system by the OS vendor, and on other brings in a custom repository and version not supported by the OS vendor.

I would be more confident if we required the end user to set manage_package_repo and version explicitly and defaulting to what the OS recommend.

Reply:
My only concern is that for some OSes the module will work out of the box and for Debian 10/11 you'll need `mongodb::globals` class as well

Reply:
But since Debian does not support MongoDB out of the box, that seems fair.   Maybe I am the only one worried by this :-)  Let's wait for other reviewers opinion about this.